### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_Buttons_Usability: The "with icon only" button is not visible permanently on the screen

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#178]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yUsable;#A11yTCS;#A11ySev4;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#FTP;#DesktopApp;#AINotInScope;#Win32;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to Button and tab to "with icon only" button check if it is permanently visible

### Actual:
The "with icon only" button is not visible permanently on the screen. It appears when we hover over it or it receives keyboard focus

### Expected:
The "with icon only" button should be visible permanently on the screen.

### User Impact:
Low vision users will find it difficult to view that if there is button present or not
